python/semantic_kernel/contents/utils/finish_reason.py

```diff
@@ -10,3 +10,9 @@ class FinishReason(str, Enum):
     CONTENT_FILTER = "content_filter"
     TOOL_CALLS = "tool_calls"
     FUNCTION_CALL = "function_call"
+    
+    # anthropic finish reasons
+    END = "end_turn"
+    MAX_TOKENS = "max_tokens"
+    STOP_SEQUENCE = "stop_sequence"
+    TOOL_USE = "tool_use"    
```

Comment:
My suggestion would be to map these semantically to each other, so stop_sequence -> stop, tool_use -> tool_calls, max_tokens -> length, end_turn -> stop, the last one I'm not sure about, but the key is to add a mapper to a const.py in anthropic to define and maintain this. cc @TaoChenOSU @moonbox3 what do you guys think?

Reply:
Sounds like a good idea to me.

Reply:
We can probably map whatever is in Anthropic to what we currently have defined. And if there is no match, we set it to None. The raw finish reason should be part of the inner content if the end user needs it. This is what is done for Google: https://github.com/microsoft/semantic-kernel/blob/main/python/semantic_kernel/connectors/ai/google/google_ai/services/utils.py#L30

Reply:
Seeing that, ideally something like this is handled with a dict!

```python
FINISH_REASON_MAPPING = {
    AnthropicFinishReason.END: FinishReason.STOP
    ....
}
```